src/components/SearchBar.jsx

```diff
@@ -16,59 +29,148 @@ export default class SearchBar extends Component {
-                type="search" 
+              <Input
+                placeholder="Search"
+                fullWidth
+                style={{ paddingLeft: '3px' }}
+                type="search"
                 onChange={event => this.setState({ value: event.target.value })}
                 value={this.state.value}
               />
-              <Button 
+
+              <Button
                 type="submit"
                 variant="contained"
                 style={styles.buttonStyle}
               >
-              Search
+                Search
               </Button>
             </form>
           </Grid>
+          
+          <Grid item xs={8}>
+            <div style={{ textAlign: 'center' }}>
+              {/*Sort feature*/}
+              <FormControl style={{ margin: '0 5px' }}>
+                <InputLabel htmlFor="sort-feature">Sort by</InputLabel>
+                <Select
+                  value={this.state.order}
+                  onChange={event =>
+                    this.setState({ order: event.target.value })
+                  }
+                  input={<Input name="sort" id="sort-feature" />}
+                >
+                  <MenuItem value="relevance">Relevance</MenuItem>
+                  <MenuItem value="date">Upload Date</MenuItem>
+                  <MenuItem value="rating">Rating</MenuItem>
+                  <MenuItem value="viewCount">View Count</MenuItem>
+                </Select>
+              </FormControl>
+
+              {/*Type*/}
+
+              <FormControl style={{ margin: '0 5px' }}>
+                <InputLabel htmlFor="type-feature">Type</InputLabel>
+                <Select
+                  value={this.state.type}
+                  onChange={this.handleType}
+                  input={<Input name="type" id="type-feature" />}
+                >
+                  <MenuItem value="video,channel,playlist">All</MenuItem>
+                  <MenuItem value="channel">Channel</MenuItem>
+                  <MenuItem value="playlist">Playlist</MenuItem>
+                  <MenuItem value="video">Video</MenuItem>
+                </Select>
+              </FormControl>
+
+              {/*Duration*/}
+
+              <FormControl style={{ margin: '0 5px' }}>
+                <InputLabel htmlFor="duration-feature">Duration</InputLabel>
+                <Select
+                  value={this.state.videoDuration}
+                  onChange={this.handleDuration}
+                  input={<Input name="duration" id="duration-feature" />}
+                  style={{ minWidth: '80px' }}
+                >
+                  <MenuItem value="any">Any</MenuItem>
+                  <MenuItem value="short">{'Short (< 4 minutes)'}</MenuItem>
+                  <MenuItem value="long">{'Long (> 20 minutes)'}</MenuItem>
+                </Select>
+              </FormControl>
+            </div>
+          </Grid>
+          <SearchResults videos={this.state.videos}/>
```

Comment:
I imagined that SearchResults component is defined with other route. but it is hard to do it from now.
anyway, I think you should hide "This is a ... " sentence during showing video result.

Reply:
This is a good point, It would be useful to have a search component use a different route. I will make a new task card and work on it a bit later